### PR TITLE
Rename `_serviceBrand` to `$serviceBrand`

### PR DESCRIPTION
**Pull request by mjbvz**
This makes it more clear that `serviceBrand` is a public (but special) property. Using `_` suggested it was a private property

### CODE REVIEW

File: src/vs/code/electron-browser/sharedProcess/sharedProcessMain.ts [hunk 1]
```diff
@@ -197,7 +197,7 @@ class SharedProcessMain extends Disposable {
 		// Lifecycle
 
 		// Product
-		const productService = { _serviceBrand: undefined, ...product };
+		const productService = { $serviceBrand: undefined, ...product };
```
Comment (mjbvz) [inline src/vs/code/electron-browser/sharedProcess/sharedProcessMain.ts:200]: Open to alternate name suggestions for this too. I know we use `$` for ipc methods today

**Review by jrieken — APPROVED** [review]
lgtm - tho there seems to be build issues. maybe not all things have been covered

**Comment by jrieken** [comment]
Also, let's hold back until we have team buy-in for doing this

**Comment by jrieken** [comment]
I actually believe this isn't needed because `_serviceBrand: undefined` doesn't make it into the outputted JS. That means there is nothing mangle can break

**Comment by mjbvz** [comment]
Closing as no longer needed

**Pull request closed without merging**